packages/core/src/lib/defaults.ts

```diff
@@ -130,7 +130,7 @@ export function resolveDefaults <TypeInfo extends BaseKeystoneTypeInfo> (config:
     },
     lists: injectDefaults(config, defaultIdField),
     server: {
```

Comment:
```suggestion
    server: {
      ...config.server,
```

In the event other fields are added but not accounted for here